### ISSUE TITLE
[New build - DEFAULT] 20w24c: More Hotfixes.

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#2413]
More Hotfixes.

Changes
- Command state preview removed, to be replaced later on.

Bugfixes
#2401 Quit to title and return to game freeze player.
#2400 Multiplayer ship spawning at same locations.
#2391 Cannot redock to an undocked port.
#2380 overlapping UI elements

**Comment by tsunamayo** [comment]
Build is live! Tell me if you still have the teleport / warp issue when entering a gravity field or a chair...

**Comment by Sullos** [comment]
Nothing yet.. so far so good! Will let you know if it comes up, but classic ways I would reproduce it, haven't caused any issues

**Comment by Sullos** [comment]
Just curious, did you change the way entities are spawned in? The game seems slower at spawning in blueprints, especially the larger they are. Whilst spawning in some station modules to continue work on them, I noticed that modules that used to spawn in nearly straight away are taking upwards of 10 seconds some times.

**Comment by tsunamayo** [comment]
No, it should have been faster since 20w23d. Maybe it is an issue with the CPU culling, it has been giving me a lot of headache already.
